### ISSUE TITLE
Media Source: <256kbps .mp3 that loads off of Google Drive using the new virtual drive will not play

### ISSUE THREAD

**Issue by BOLL7708** [obsproject/obs-studio#4482]
### Operating System Info

Windows 10

### OBS Studio Version

26.1.2

### OBS Studio Log URL

https://obsproject.com/logs/V09YmfywxiskWlri

### Expected Behavior

The Media Source with mp3 file should play when switching to that scene or pressing the restart button.

### Current Behavior

The Media Source with mp3 does not play, and cannot be manually restarted either. 
Sometimes the first media source in the current scene collection will play, but only once, and only that one.

### Steps to Reproduce

1. Make a random .mp3 using: https://ttsmp3.com/ they end up as 48kbps 20500Hz mono files.
2. Install the latest version of [Google Drive for Desktop](https://support.google.com/drive/answer/7329379#zippy=%2Cdownload-install-google-drive-for-desktop) that mounts a virtual drive: 
3. Put the generated .mp3 file on your Google Drive virtual drive.
4. Go into OBS and add a Media Source, point it towards the .mp3 file that is in the Google Drive.
5. Try to play the media source, or have it trigger by switching scenes.

### Anything else we should know?

Switching file format to .wav has it working, so it appears it's the mp3 decoder specifically that breaks down in this case. I tested this on the version pf OBS in this submission, a friend tested it with both and older version and the v27 RC1 version, issue persists through all.
Simply moving the files to a physical drive resolves it, but would be nice if it worked out of the Google Drive too, of course.

Edit: Further [testing](https://github.com/obsproject/obs-studio/issues/4482#issuecomment-816813793) has shown that resaving at 256 kbps or higher bitrate makes the files work, values below 256 appear to be the what causes trouble when combined with Google Drive.

**Comment by derrod** [comment]
Can't reproduce, are the files marked as available offline?

I would recommend against using it without the files being marked as available offline, otherwise you might run into rate limits relatively quickly and are locked out of accessing *any* of your files for a few hours.

**Comment by BOLL7708** [comment]
The files have indeed been marked as `Available offline`. As it works for you I tried launching OBS as administrator, figured that could be a differentiator, but nope. 🤔 

Now I'm starting to wonder if it has to do with symbols in the file path, but then it worked well with the old Backup & Sync and it's the same internal path now.

It's also curious how my friend had it to behave the same way, even on a different PC with a different folder structure. He also used files generated off of ttsmp3.com though... guess I should try a few other ones and see if they behave the same way.

Another detail is that the playback works just fine if I simply copy/move the files to a folder outside of Google Drive, which is why I expect Google Drive to be the issue 👀 but then it's only for .mp3 files so possible the decoder.

The obvious solution is to just convert to .wav or similar, still, if there actually is an issue here it would be nice to see it resolved ,too.

**Comment by derrod** [comment]
I tried an mp4 file from ttsmp3 as well now, with that I can reproduce it. Not sure what's different with those mp3 files that's causing issues.

**Comment by BOLL7708** [comment]
Did some experimentation just now. Taking one of the problematic files as a base, and seeing what fixed it, this is what I saw. Only listing what I changed, other properties would be bits per sample: 32. Stereo or mono did not make any apparent difference, all these files tested are mono.
* Original, bitrate: 48 kbps, sample rate: 22050 Hz - broken
* Sample rate: 24000 - broken
* Bitrate: 56 kbps, sample rate: 44100 Hz - broken
* Bitrate: 64 kbps, sample rate: 44100 Hz - broken
* Bitrate: 80 kbps, sample rate: 44100 Hz - broken
* Bitrate: 96 kbps, sample rate: 44100 Hz - broken
* Low quality VBR (like 96 kbps CBR), sample rate: 44100 Hz - broken
* Bitrate: 128 kbps, sample rate: 44100 Hz - broken
* Bitrate: 192 kbps, sample rate: 44100 Hz - broken
* Bitrate: 256 kbps, sample rate: 44100 Hz - works
* Bitrate: 320 kbps, sample rate: 44100 Hz - works

From this it appears as if low bitrate .mp3 files off the new Google Drive causes issues 🤔

**Comment by Fenrirthviti** [comment]
Is this reproduced when not using a Google Drive network location?

If it works fine with all local file, just to be honest here, this is an effectively something that Google Drive would need to fix, as it introduces some kind of issue with the files.

**Comment by Fenrirthviti** [comment]
Closing as stale. If this is still an issue please open a new issue with updated reproduction steps on the latest version of OBS.
